docs/1-Using-Fleet/3-REST-API.md

```diff
@@ -3068,3 +3033,83 @@ Modifies the osquery options configuration set in Fleet.
 ```
 
 ---
+
+## Fleet UI
+
+### Get hosts' table columns
```

Comment:
I wonder if we should make this a general endpoint for storing user UI preferences? I can't think of any off the top of my head (can you?) but I imagine that this won't be the last data we want to store about user UI preferences.

Reply:
I agree with your general endpoint suggestion. One example future preference I can think of: storing is an open v. closed state if we choose to implement a collapsable accordion on future iterations of the _Host details_ page. (maybe)

My latest commit includes changes with a possible implementation for this general endpoint concept.